kotlin-inject-compiler/test/src/test/kotlin/me/tatarka/inject/test/WarningTest.kt

```diff
@@ -35,6 +35,7 @@ class WarningTest {
 
     @ParameterizedTest
     @EnumSource(Target::class)
+    @Suppress("UNUSED_PARAMETER")
     fun warns_on_implicit_assisted_params(target: Target) {
         val projectCompiler = ProjectCompiler(Target.KAPT, workingDir)
```

Comment:
Does this test need to be parameterized? It looks like `KAPT` is hardcoded in.

Reply:
good catch! It might have been a copy/paste error. I'd say try it and if it's failing for ksp for some reason we can revisit that in a different pr.

Reply:
Works with ksp :tada: